resources/lib/ipwww_video.py

```diff
@@ -740,47 +742,41 @@ def Search(search_entered):
     ScrapeEpisodes(NEW_URL)
 
 
+
 def AddAvailableLiveStreamItem(name, channelname, iconimage):
     """Play a live stream based on settings for preferred live source and bitrate."""
-    stream_bitrates = [9999, 345, 501, 923, 1470, 1700, 2128, 2908, 3628, 5166]
-    if int(ADDON.getSetting('live_source')) == 1:
-        providers = [('ak', 'Akamai')]
-    elif int(ADDON.getSetting('live_source')) == 2:
-        providers = [('llnw', 'Limelight')]
-    else:
-        providers = [('ak', 'Akamai'), ('llnw', 'Limelight')]
+    stream_bitrates = [9999, 576, 974, 1732, 3000, 5308] 
```

Comment:
Please keep the original bitrate, more info in a comment on the changes to settings.xml.